package.json

```diff
@@ -34,6 +34,7 @@
     "dev": "NODE_ENV=development BROWSERSYNC_OPEN=true gulp dev",
     "docs:mod": "node tasks/mod-extractor.js",
     "lint:components": "node ./tasks/packageLint.js",
+    "lint:styles": "stylelint ./components/**/*.css",
```

Comment:
+1

Reply:
Perhaps we should hook this in through postcss so that another command is not necessary: https://stylelint.io/user-guide/postcss-plugin?

Reply:
We wanted to give the users to see lint warnings without doing the build.

Reply:
I think I'd prefer to leave it as a separate command for now. Additionally, I noticed this sentence in that link:

> However, we recommend using the CLI or Node.js API (directly or via an integration) as they provide better reporting.

I'm cool with keeping it as a separate command if you all are.